src/Common/tools/add-extension.ps1

```diff
@@ -49,7 +49,7 @@
            throw "Extension bundle module path does not exist $extensionModulePath"
         }
 
-        $latest = Get-ChildItem -Path $extensionModulePath | Sort-Object LastAccessTime -Descending | Select-Object -First 1
+        $latest = Get-ChildItem -Path $extensionModulePath | Sort-Object name -Descending | Select-Object -First 1
```

Comment:
it should pick the latest one, isn't it?

Reply:
it was going to the previous version in my case if I accessed and file in the old folder , so getting the folder by name is more guaranteed